spec/features/airlines/show_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'Airlines show', type: :feature do
+  before(:each) do
+    @airline_1 = Airline.create!(name: 'Frontier')
+    @airline_2 = Airline.create!(name: 'Delta')
+    @airline_3 = Airline.create!(name: 'Southwest')
+    
+    @flight_1 = @airline_1.flights.create!(number: '1727', date: '08/03/24', departure_city: 'Denver', arrival_city: 'Louisville')
+    @flight_2 = @airline_2.flights.create!(number: '1834', date: '04/03/24', departure_city: 'New York', arrival_city: 'San Francisco')
+    @flight_3 = @airline_3.flights.create!(number: '2674', date: '11/07/24', departure_city: 'Chicago', arrival_city: 'Miami')
+
+    @passenger_1 = @flight_1.passengers.create!(name: 'Joe', age: 7)
+    @passenger_2 = @flight_1.passengers.create!(name: 'Kaelin', age: 14)
+    @passenger_3 = @flight_2.passengers.create!(name: 'Karl', age: 27)
+    @passenger_4 = @flight_3.passengers.create!(name: 'Lito', age: 45)
+    @passenger_5 = @flight_3.passengers.create!(name: 'Lance', age: 23)
+  end
```

Comment:
Your implementation of US 3 is not correct. In order to properly test the US, you should have adult passengers on multiple flights, and in the view, they should only be showing up once. Right now you do not have the same adult passenger on multiple flights.